lib/src/widgets/image_source_sheet.dart

```diff
@@ -25,7 +26,7 @@ class ImageSourceBottomSheet extends StatelessWidget {
   /// Callback when an image is selected.
   ///
   /// **Note**: This will work on web platform whereas [onImageSelected] will not.
-  final Function(Image) onImage;
+  final Function(Uint8List) onImage;
```

Comment:
`Uint8List` give developers more options to manipulate / store etc. Image is only useful for display purposes.